src/provider/CoinbaseWalletProvider.ts

```diff
@@ -50,7 +50,7 @@ const HAS_CHAIN_OVERRIDDEN_FROM_RELAY = "HasChainOverriddenFromRelay";
 export interface CoinbaseWalletProviderOptions {
   chainId?: number;
   jsonRpcUrl: string;
-  qrUrl: string | null;
+  qrUrl?: string | null;
```

Comment:
isn’t it supposed to be something like
```suggestion
  qrUrl?: string = undefined;
```
and come as last param or something?

Reply:
nvm. confirmed it works. i think i was confused with optional function param.

Reply:
Does it need `null` at all?